one-ui/ui/api/swagger/mocks.json

```diff
@@ -1,9 +1,8 @@
 {
   "swagger": "2.0",
   "info": {
-    "description": "This is a sample of a DataHub Swagger document defining endpoints to be mocked",
-    "version": "1.0.0",
-    "title": "Data Hub Curation API",
+    "version": "5.3.0-SNAPSHOT",
```

Comment:
We need to include this File in the task "updateVersion" in the main project which will update the versions whenever we release.
@rjrudin  Are we going to release these API's in future?

Reply:
We don't have any requirements for the version yet - I just figured "5.3.0-SNAPSHOT" was more meaningful than "1.0.0".

Reply:
I added mocks.json to the task in ./build.gradle, just in case.